cuebot/src/main/java/com/imageworks/spcue/dao/criteria/postgres/Criteria.java

```diff
@@ -17,27 +17,19 @@
 
 package com.imageworks.spcue.dao.criteria.postgres;
 
+import com.google.common.collect.ImmutableList;
+import com.imageworks.spcue.dao.criteria.CriteriaInterface;
+import com.imageworks.spcue.dao.criteria.Phrase;
+import com.imageworks.spcue.dao.criteria.Sort;
+import com.imageworks.spcue.grpc.criterion.*;
```

Comment:
We have tried to avoid wildcard imports, do you have any reason to prefer them here?

Reply:
Sorry I didn't have the style setup properly in IDEA, I can fix.